tests/sentry/integrations/github/tasks/test_pr_comment.py

```diff
@@ -526,8 +527,8 @@ def test_comment_workflow_api_error(self, mock_metrics, mock_issues):
 
         with pytest.raises(ApiError):
             github_comment_workflow(self.pr.id, self.project.id)
-            assert cache.get(self.cache_key) is None
-            mock_metrics.incr.assert_called_with("github.pr_comment.api_error")
+        assert cache.get(self.cache_key) is None
```

Comment:
These were not previously run because the assertions occur after the exception is raised, which is then absorbed by the pytest.raises context manager.